README.md

```diff
@@ -31,7 +31,9 @@ steps:
       secret: ${{ github.TOKEN }}
       approvers: user1,user2
       minimum-approvals: 1
+      issue-title: "Deploying v1.3.5 to prod from staging."
```

Comment:
Nit: People probably wouldn't put a period at the end of their title, so we should probably remove it for the example so it isn't misleading.

```suggestion
      issue-title: "Deploying v1.3.5 to prod from staging"
```